.travis.yml

```diff
@@ -1,64 +1,45 @@
-language: generic
+language: python
+dist: xenial
+python: 3.6
 
-sudo: false
+notifications:
+  email: false
 
-cache:
-    directories:
-        - ${HOME}/miniconda3
+services:
+  - docker
 
-env:
-    global:
-        - secure: "bF1Ljix0pDNBBjgewrOxxNHJDEvlWDT5v5KckSq/Wl0v9QMAG5Fiilj0hPHVQrWoGz7F3xPU5KeprjQcfTo2UB7qPGDOVsovafbUTZ+0o6kc2mVLApYsRvlhUPX8tsRCrq6Zo874buMzbDPyAo3nnl3lkkWQ2Dh+gA7b7er9xS0="
+sudo: required
 
-# Travis has limited OSX resources
-# Only do OSX on Python 3.6
-matrix:
-    include:
-      - os: linux
-        env: CONDA_PY=36 BUILD_DOC=1
-      - os: linux
-        env: CONDA_PY=37
-      - os: osx
-        env: CONDA_PY=36
-             MACOSX_DEPLOYMENT_TARGET=10.9
+env:
+  global:
+    - DOCKER_IMAGE=pywr/manylinux1_x86_64-glpk
```

Comment:
Where does the Dockerfile for this live?

Reply:
It is on my machine atm. Where do you think it should live? We have a `pywr-docker` repo. I guess it should go there.

Reply:
It could live in the `travis/` directory in this repo.